### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by dcr-stripe**
Codegen for openapi v173.
r? @pakrym-stripe
cc @stripe/api-libraries

## Changelog
* Add support for new value `exempted` on enums `ChargePaymentMethodDetailsCardThreeDSecureResult` and `SetupAttemptPaymentMethodDetailsCardThreeDSecureResult`
* Add support for `CustomerBalance` on `CheckoutSessionPaymentMethodOptionsParams` and `CheckoutSessionPaymentMethodOptions`

### CODE REVIEW

File: checkout_session.go [hunk 1]
```diff
@@ -285,6 +285,63 @@ const (
 	CheckoutSessionPaymentMethodOptionsCardSetupFutureUsageOnSession  CheckoutSessionPaymentMethodOptionsCardSetupFutureUsage = "on_session"
 )
 
+// The desired country code of the bank account information. Permitted values include: `DE`, `ES`, `FR`, `IE`, or `NL`.
+type CheckoutSessionPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountry string
+
+// List of values that CheckoutSessionPaymentMethodOptionsCustomerBalanceBankTransferEUBankTransferCountry can take
+const (
```
Comment (pakrym-stripe) [inline checkout_session.go:292]: Should we remove these?

**Review by pakrym-stripe — APPROVED** [review]
One question.

File: checkout_session.go (hunk 1 above)
Comment (dcr-stripe) [reply checkout_session.go:292]: Yep. Good catch.

Comment (dcr-stripe) [reply checkout_session.go]: Done.

**Comment by dcr-stripe** [comment]
r? @pakrym-stripe 

PTAL

**Review by pakrym-stripe — APPROVED** [review]

**Pull request merged**